src/app/containers/ChartbeatAnalytics/index.jsx

```diff
@@ -41,6 +41,7 @@ const ChartbeatAnalytics = ({ data }) => {
     service,
     sendCanonicalChartbeatBeacon,
     isCanonicalAndEnabled,
+    chartbeatConfig,
```

Comment:
This change is unexpected?

Reply:
Was a prettier autofix - does seem legitimate as it fixes an eslint warning - `34:6  warning  React Hook useEffect has a missing dependency: 'chartbeatConfig'. Either include it or remove the dependency array  react-hooks/exhaustive-deps` - can remove it though as in scope for this PR

Reply:
OK makes sense now though I'd still leave it to be fixed separately!

Reply:
👍 removed